### DISCUSSION TITLE
Bonsai is hiring

### DISCUSSION THREAD

**Discussion by glopesdev** [bonsai-rx/bonsai#1504] [General]
Following the release of the new Bonsai 2.8, we are excited to share below several open positions for working directly with our team on the development of the Bonsai programming language, spanning different roles and host institutions:

- [Research Software Engineer, Experiments @ NeuroGEARS | London, UK](https://neurogears.org/jobs/20164152)
- [Application Software Engineer, .NET @ NeuroGEARS | (London, Lisbon or Remote options)](https://neurogears.org/jobs/20230801)
- [Technical Writer, Outreach @ NeuroGEARS | (London, Lisbon or Remote options)](https://neurogears.org/jobs/20230802)
- [Research Software Engineer @ UCL (Neuroscience/Machine Learning) | London, UK](https://www.ucl.ac.uk/work-at-ucl/search-ucl-jobs/details?jobId=13635&jobTitle=Research+Software+Engineer+%28Neuroscience%2FMachine+Learning%29)

This year will be special for our community with a renewed investment and interest from several organizations in establishing Bonsai as a foundation for experimental control and data acquisition in Neuroscience. We are extremely excited with this level of support and we are determined to make the most out of this unique opportunity.

If you ever wondered whether there was a way to be supported and get involved to learn more about the language internals, work in cutting edge science in unusual positions, tinker and teach on a variety of topics, or contribute directly to open-source science, this post is for you!

Even if you think you might not have all the skills listed in these positions, or come from different backgrounds, but would still like to get involved, we encourage you to apply or simply get in touch with us.

**Comment by mrakgr** [comment]
Saw your post on HN and sent an email. Whether it be PL development, 3d art or reactive functional programming, I do have all the skills needed for the remote position, and am hoping that by posting in this thread, it will insure my application against being overlooked in the deluge. Hope to hear from you.
